src/utils/functions.js

```diff
@@ -0,0 +1,9 @@
+export function containerInformation(search, i) {
```

Comment:
If we are going to have a utils directory, have a file for each function named the same as the function!